bosh-stemcell/spec/stemcells/centos_7_spec.rb

```diff
@@ -9,4 +9,18 @@
       it { should contain('centos') }
     end
   end
+
```

Comment:
we can move this to shared example for all centos/rhel stemcells to avoid duplication: stemcell_centos_rhel_7_shared_examples.rb